confluent_kafka/src/confluent_kafka.c

```diff
@@ -880,13 +879,11 @@ static int stats_cb(rd_kafka_t *rk, char *json, size_t json_len, void *opaque)
  * Clear Python object references in Handle
  */
 void Handle_clear (Handle *h) {
-	if (h->error_cb) {
+	if (h->error_cb) 
```

Comment:
nitpick: There's a trailing whitespace after ).  This might seem silly but it will show up as a red blob in editors and git gui :)